roles/ckan/tasks/deploy.yml

```diff
@@ -75,19 +74,19 @@
     - db.yaml
   when: fjelltopp_env_type == 'local'
 
-- name: Check if DB is alive [local] (10 tries, 5sec interval)
-  community.postgresql.postgresql_ping:
-    db: ckan
-    login_host: "{{ ckan_fqdn }}"
-    login_user: "{{ rds_admin_username }}"
-    login_password: "{{ ckan_postgres_password }}"
-  when: (fjelltopp_env_type == 'local')
-  register: db_ping_result
-  retries: 10
-  delay: 5
-  until: db_ping_result is defined and db_ping_result.get('is_available')
-  failed_when: db_ping_result is not defined or not db_ping_result.is_available
-  # For this check to work psycopg2 must be installed for the venv python referenced from your inventory
+# - name: Check if DB is alive [local] (10 tries, 5sec interval)
+#   community.postgresql.postgresql_ping:
+#     db: ckan
+#     login_host: "{{ ckan_fqdn }}"
+#     login_user: "{{ rds_admin_username }}"
+#     login_password: "{{ ckan_postgres_password }}"
+#   when: (fjelltopp_env_type == 'local')
+#   register: db_ping_result
+#   retries: 10
+#   delay: 5
+#   until: db_ping_result is defined and db_ping_result.get('is_available')
+#   failed_when: db_ping_result is not defined or not db_ping_result.is_available
+#   # For this check to work psycopg2 must be installed for the venv python referenced from your inventory
```

Comment:
This didn't work, probably didn't have psycopg2 installed or something. Will deal with another time